project-progression-template.md

```diff
@@ -4,24 +4,47 @@
 3.  For any questions or comments on this process please email info@confidentialcomputing.io
 
 ## Project Proposal Template
-1.  Name of Project
-2.  Project Description (what it does, why it is valuable, origin and history)
-3.  How does this project align with the Consortium's [Mission Statement](README.md)
-4.  High level assessment of project synergy with existing projects under the CCC, including how the project compliments/overlaps with existing projects, and potential ways to harmonize over time. Responses may be included both inline and/or in accompanying documentation.
-5. Describe the [Trusted Computing Base (TCB)](https://en.wikipedia.org/wiki/Trusted_computing_base) of the project. If the project supports multiple environments please describe each TCB. Also identify if dependencies of other project (both CCC or non-CCC) TCBs are taken.
```

Comment:
@spjohnso feel free to move it where ever you see fit, or work into one of the other questions

Reply:
@annabellegoth2boss It's a little bit awkward to modify someone else's PR. Could you please re-add / undelete point 5?